ci/builders/standalone/linux_android_emulator.json

```diff
@@ -0,0 +1,46 @@
+{
+    "drone_dimensions": [
+        "device_type=none",
+        "os=Linux"
+    ],
+    "gn": [
+         "--android",
+         "--android-cpu=x64",
+         "--no-lto"
+    ],
+    "name": "android_debug_x64",
+    "ninja": {
+        "config": "android_debug_x64",
+        "targets": [
+            "flutter/shell/platform/android:flutter_shell_native_unittests",
+            "flutter/testing/scenario_app"
+        ]
+    },
+    "tests": [
+        {
+            "language": "python3",
+            "name": "Android Unit Tests",
+            "contexts": [
```

Comment:
I don't see this field documented in https://github.com/flutter/engine/blob/main/ci/builders/README.md.

Reply:
https://github.com/flutter/flutter/issues/129187 is tracking the documentation.